index.html

```diff
@@ -0,0 +1,129 @@
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <!-- Main Content Section -->
+        <main>
+        <section class="hero-section">
+            <div class="hero-div">
+                <h1 class="thin-text">"Hello! Sharing Knowledge around the Globe"
+                    <h2 class="thick-text">Awareness Summit about web development 2023</h2>
+                </h1>
+                <span class="description-about-summit">
+                     A joyful and informative enviroment for sharing creative ideas about the advancements of technology.
+                </span>
+                <span class="summit-date">
+                    2023.10.15(FRI)
+                </span>
+                <span class="summit-location">
+                    @ABC Avenue NY
+                </span>
+            </div>
+        </section>
+                        <!-- main program section -->
+        <section 
+        class="main-program-section">
+            <div class="main-program-div">
+                 <h2>Main Program</h2>
+                <div class="main-program-item-container">
+                    <article class="main-program-item">
+                        <img src="./assets/imgs/program_icon_01 1.png" alt="">
+                        <h3>Lecture</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </article>
+                    <article class="main-program-item">
+                        <img src="./assets/imgs/program_icon_02 1.png" alt="">
+                        <h3>Exhibition</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </article>
+                    <article class="main-program-item">
+                        <img src="./assets/imgs/program_icon_03 1.png" alt="">
+                        <h3>Forum</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </article>
+                    <article class="main-program-item">
+                        <img src="./assets/imgs/program_icon_04 1.png" alt="">
+                        <h3>Workshop</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </article>
+                    <article class="main-program-item">
+                        <img src="./assets/imgs/program_icon_05 1.png" alt="">
+                        <h3>Ignite</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </article>
+                </div>
```

Comment:
- [ ] Nice work adding the images :clap: However as suggested by the previous reviewer, please stick to the Figma design as much as possible. Therefore, I suggest you reduce the margins on the mobile version as shown in the image below.
### Your design :point_down: 
![Screenshot from 2023-06-27 14-22-31](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/91142432/a5fe9de5-c92d-47ff-a542-ded0d3b0af62)
### Figma's design :point_down: 
![249111519-d9e6d733-a19f-49e7-974d-fb8a27e0bef6](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/91142432/6935baf0-daa9-419c-bf90-085addb11244)

Reply:
**OPTIONAL** - You have done well so far, as much as you are required to personalize the content of your page, you are to stick to the design template in terms of spacing, colors, background colors etc.  Your program cards are not styled like the design template, please fix this 🙏 
Your project:
-- Please fix this issue for both mobile and desktop version 🙏 
![wrgawr](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/73607512/1fc2e789-bafb-4827-95a4-f80dfc005caa)
Design template:
![thsethseth](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/73607512/5a8da893-af46-4208-bd06-4ec992c53f52)